deploy/helm/sumologic/templates/otelcol-service.yaml

```diff
@@ -10,18 +10,26 @@ spec:
   selector:
     app: {{ printf "%s-otelcol" (include "sumologic.labels.app" .) }}
   ports:
-  - name: opencensus # Default endpoint for Opencensus receiver.
-    port: 55678
-    protocol: TCP
-    targetPort: 55678
+  - name: jaeger-sampling # Default endpoint for Jaeger Sampling (if enabled)
+    port: 5778
```

Comment:
Does it listen on `0.0.0.0`?

Reply:
Good catch. Right now, it's just a first step towards enabling remote sampling. Sampling itself requires separate entry in the configuration and we'll need to provide the details in the guide. So the purpose of that (as for now) is to just make less changes to `values.yaml`